Code/Source/RobotImporter/RobotImporterWidget.cpp

```diff
@@ -83,6 +83,13 @@ namespace ROS2
                     ReportError("User cancelled");
                     return;
                 }
+
+                if (IsDependencyCyclical(prefabName.Filename()))
+                {
+                    ReportError("Cyclical dependency detected.\nPlease choose another prefab to import or exit the focus mode.");
```

Comment:
I think we should use more user-friendly descriptions about what is wrong and what to do.

Maybe something like:
```suggestion
                    ReportError("Selected URDF model is currently being edited. Exit prefab edit mode and try again.");
```
?

Reply:
I mentioned cyclical dependency basing on the O3DE notification which shows up when you try to manually instantiate prefab inside itself:
![image](https://user-images.githubusercontent.com/111358112/199738185-688a0f0b-28c7-4411-9d2a-1fb786baca83.png)

Reply:
It is ok that is a cyclical dependency, but we should prompt users with a more user-friendly message.

Reply:
I changed the error text.